internal/commands/util/remediation_test.go

```diff
@@ -71,6 +72,13 @@ func TestRemediationKicsCommandEngineFlag(t *testing.T) {
 	assert.Assert(t, err == nil, "Remediation command must pass")
 }
 
+func TestRemediationKicsCommandInvalidEngine(t *testing.T) {
+	cmd := RemediationKicsCommand()
+	abs, _ := filepath.Abs(kicsFileValue)
+	err := executeTestCommand(cmd, resultsFileFlag, resultFileValue, kicsFileFlag, abs, engineFlag, invalidEngineValue)
+	assert.Assert(t, err != nil, "Please verify if engine is installed and running")
```

Comment:
I would just suggest to use the constant invalidEngineMessage from remediation.go so that if the base message changes for any reason, we don't need to fix it in the tests as well

Reply:
Thanks, will do that !